general/linking.md

```diff
@@ -11,13 +11,13 @@ More specifically:
 -   **If linking to a specific _line_:** _Always_ use a fixed link.
 -   **If linking to a specific _file_:**
     -   **If it is _possible_ that the file might be renamed:** (eg. _any_ source code file) Use a fixed link.
-    -   **If it is _extremely unlikely_ that a file will be renamed:** (eg. a `package.json` or a _top-level_ `README.md`) Don't use a fixed link.
+    -   **If it is _extremely unlikely_ that a file will be renamed:** (eg. a _top-level_ `package.json` or a _top-level_ `README.md`) Don't use a fixed link.
```

Comment:
![ok](https://user-images.githubusercontent.com/7074/95487112-2e358000-0994-11eb-891a-c4c9efd0b26c.gif)

Reply:
You know... Just in case...